extension-jaxrs/src/test/java/io/smallrye/openapi/runtime/scanner/ApiResponseTests.java

```diff
@@ -201,6 +206,7 @@ public void deletePet(@PathParam("id") String id) {
         @Produces(MediaType.APPLICATION_JSON)
         @APIResponse(responseCode = "200")
         @APIResponse(responseCode = "400", description = "Description 400")
+        @APIResponseSchema(value = ServerError.class, responseDescription = "Internal Server Error!", responseCode = "500")
```

Comment:
The correctness of this test relies on the single `!` character, it might be better to use a more distinctive string as the description.

Reply:
Good suggestion, I've updated the test.